merlin-server/constraints-dsl-compiler/src/libs/constraints-edsl-fluent-api.ts

```diff
@@ -80,6 +80,26 @@ export class Windows {
     this.__astNode = expression;
   }
 
+    /**
+   * Produces windows for each activity present in the plan and belonging to one of the activity types passed
+   *
+   * @param activityTypes the activity types
+   */
+  public static DuringAny(...activityTypes: Gen.ActivityType[]) : Windows {
+    return Windows.Or(
+        ...activityTypes.map<Windows>((activityType) => Windows.During(activityType))
+    );
+  }
+
+  /*
+  * Produces windows for each activity of the argument type present in the plan
+  *
+  * @param activityType the argument type
+  */
+  public static During(activityType: Gen.ActivityType) : Windows {
+    return Spans.ForEachActivity(activityType, (activity) => activity.span()).windows();
+  }
+
```

Comment:
These can be combined into a single `During(...ActivityType)` function because `DuringAny` can be called with one argument.